cluster/examples/kubernetes/nfs/pvc.yaml

```diff
@@ -2,10 +2,11 @@ apiVersion: v1
 kind: PersistentVolumeClaim
```

Comment:
it feels like we need an updated example/documentation for this dynamic provisioning experience.  Can you add some user facing documentation that demonstrates how they can use this feature in a streamlined easy to understand way?

Reply:
yeah sure.